### ISSUE TITLE
Collection summary not synced with collection view.

### ISSUE THREAD

**Issue by AnikaErceg** [Codewars/codewars.com#892]
Even after a page refreshes and waiting a couple of hours, "Number of katas" (and "Updated at") doesn't sync between collection summary and actual collection view. 
Both screenshots were taken at the same time.
**Collection summary:**
![1](https://cloud.githubusercontent.com/assets/26693599/25004034/90ac3bea-2052-11e7-848d-252fad260c09.jpg)
**Collection view:**
![2](https://cloud.githubusercontent.com/assets/26693599/25004055/ae226cf8-2052-11e7-84b1-851e90abeae6.jpg)

Problem occurs even if I view them logged out and/or on my phone (mentioning this in case this means something).

**Comment by DonaldKellett** [comment]
I've noticed this with Collections in general ever since they were released on Codewars - it usually takes at least a few days before all pages related to said collection(s) are fully synced.  For example, when I initially add a Description for a collection and click "Save", the collections page still says that a Description is yet to be added.  Subsequent refreshes and logging in/out of Codewars are useless in speeding up the sync process.  Only after a few days when I check the Collection again do I see the Description as expected.  The same goes for any other action on a Collection - reordering, deleting, etc. - there just appears to be a standard lag for at least a few days which is sometimes a bit annoying as I cannot immediately inspect the changes that I made to a Collection.
